pom.xml

```diff
@@ -13,17 +13,12 @@
   <description>Adds the ability to wrap your job with pre- and post- build steps which are only executed when a manual release build is triggered.</description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Release+Plugin</url>
 
-  <build>
-    <plugins>
-      <plugin>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5.1</version>
-        <configuration>
-          <goals>deploy</goals>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+  <properties>
+    <jenkins.version>1.609.3</jenkins.version>
```

Comment:
It's debatable. My previous instance still work on 1.509.4, @olivergondza uses 1.580.x IIRC.
For enterprise installations it's common to be behind the trends due to [stabilization/migration testing] efforts. 

So I would ask the plugin maintainer